src/specs/data_structures.md

```diff
@@ -268,6 +268,7 @@ Merkle trees are used to authenticate various pieces of data across the Celestia
 Binary Merkle trees are constructed in the same fashion as described in [Certificate Transparency (RFC-6962)](https://tools.ietf.org/html/rfc6962), except for using [a different hashing function](#hashing). Leaves are hashed once to get leaf node values and internal node values are the hash of the concatenation of their children (either leaf nodes or other internal nodes).
 
 Nodes contain a single field:
+
```

Comment:
newlines are needed prior to a table to display correctly

Before | After
--- | ---
<img width="755" alt="Screen Shot 2022-07-31 at 11 47 08 AM" src="https://user-images.githubusercontent.com/3699047/182034416-a3f508e9-e75f-4f82-8328-ad666593874a.png"> | <img width="759" alt="Screen Shot 2022-07-31 at 11 47 36 AM" src="https://user-images.githubusercontent.com/3699047/182034424-6acdc97a-fe69-40c6-aa68-87147969b88c.png">